ws-nextjs-app/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "description": "Simorgh - Next.JS",
   "scripts": {
-    "setupDevEnv": "cp ../envConfig/local.env .env",
+    "setupDevEnv": "cp ../envConfig/local.env .env && mkdir -p public && cp -r ../public/. public",
```

Comment:
Copies across the `public` folder from the Express app into the Next.js app. This copies across the `manifest.json` files for the service, so stops annoying console warnings in local development, but importantly copies across the existing `sw.js` file so that we don't need to repeat it for the Next.js app when working locally.